src/bitExpert/Adroit/Responder/Responder.php

```diff
@@ -30,7 +32,7 @@
      *
      * @param Payload $payload
      * @param ResponseInterface $response
-     * @return ResponseInterface
+     * @return ResponseInterface|null
```

Comment:
Can you please provide a use case where it makes sense to let a responder return null instead of a response? I think this should never happen. If you agree, please change to return type ResponseInterface